Machine.Specifications.ReSharper.Provider.10/MspecTestArtefactExplorer.cs

```diff
@@ -11,14 +12,14 @@ namespace Machine.Specifications.ReSharperProvider
     public class MspecTestArtefactExplorer : DotNetTestArtefactExplorer<MspecTestProvider>
     {
         public MspecTestArtefactExplorer(
-            MspecTestProvider provider, 
-            IDotNetCoreSdkResolver sdkResolver, 
-            IDotNetCoreUnitTestServerFactory serverFactory, 
-            MspecTestElementMapperFactory mapperFactory, 
-            IProcessRunnerManager processRunnerManager, 
-            IDotNetTestCaseMap testCaseMap, 
-            ILogger logger) 
-            : base(provider, sdkResolver, serverFactory, mapperFactory, processRunnerManager, testCaseMap, logger)
+            [NotNull] MspecTestProvider provider, 
+            [NotNull] IDotNetCoreSdkResolver sdkResolver, 
+            [NotNull] IJsonBasedUnitTestServerFactory serverFactory, 
+            [NotNull] IDotNetTestElementMapperFactory mapperFactory, 
```

Comment:
The mapper factory must be the `MspecTestElementMapperFactory` or it won't resolve our instance.